### DISCUSSION TITLE
Activity Feed Presets and Bookmarks Missing

### DISCUSSION THREAD

**Discussion by w0kyj** [directus/directus#24908] [Feature Requests]
### Describe the Bug

Directus Activity Log -- The Activity Feed page doesn't have app support for presets and bookmarks.

Admin settings for Presets & Bookmarks allow selection of the Directus Activity collection but do not show on the Activity Log page.

### To Reproduce

Missing Bookmark Feature next to "Activity Feed" and only the precreated filters available in the left navigation tree:
<img width="1381" alt="Screen Shot 2022-09-29 at 1 41 13 PM" src="https://user-images.githubusercontent.com/40695629/193126909-858c5e62-15d5-405f-be2a-104207038869.png">

Admin Presets & Bookmarks allows selection of the Directus Activity collection, but do not show on the Activity Log page above:
<img width="1373" alt="Screen Shot 2022-09-29 at 1 44 27 PM" src="https://user-images.githubusercontent.com/40695629/193127302-1c991bb2-77d4-4986-a9bc-4c4d0ab18576.png">

### Errors Shown

N/A

### What version of Directus are you using?

9.18.1

### What version of Node.js are you using?

Directus Cloud

### What database are you using?

Directus Cloud(Postgres)

### What browser are you using?

Chrome

### How are you deploying Directus?

Directus Cloud

**Comment by azrikahar** [comment]
Out of curiosity as `Directus Files` and `Directus Users` are also technically selectable in Presets & bookmarks, is this issue meant as "Files and Users works but not Activity" (which I was not able to achieve, but wanted to ask to be sure), or it just so happens to be encountered and reported for Activity?

Activity seems to have been added back in beta https://github.com/directus/directus/commit/af25b83c7630fc791b67c2f21a638e0d5b8019d5, so not sure (and curious to know) when did it became missing, if it did work before.

**Comment by Nitwel** [comment]
I'm pretty sure it never became missing but was added to the whitelist in order to be able to configure the default preset and that's it.

So I'm not sure if we even want to add bookmark support to users, files and activity.

**Comment by w0kyj** [comment]
This issue was specifically for Activity, but reviewing the others:
- Users should support this as well -- we have clients with 100K+ users, so bookmarks support makes sense
- Files would be nice to have, but with the folder structures, this one is a bit more complex.
    - Open to discussion on this one and lowest priority of these three

If we need additional issues for the other two, let me know and I will open them.

**Comment by rijkvanzanten** [comment]
Linear: [ENG-174](https://linear.app/directus/issue/ENG-174/activity-feed-presets-and-bookmarks-missing)
